plugins/google/pkg/app/transform.go

```diff
@@ -2,23 +2,27 @@
 package app
 
 import (
+	"context"
 	"os"
+	"time"
 
 	"github.com/alecthomas/kong"
-	"github.com/aserto-dev/ds-load/sdk/plugin"
+	"github.com/aserto-dev/ds-load/sdk/transform"
 )
 
 type TransformCmd struct {
-	Template     string `name:"template" short:"t" env:"DS_TEMPLATE_FILE" help:"transformation template file path" type:"path" optional:""`
-	MaxChunkSize int    `name:"max-chunk-size" env:"DS_MAX_CHUNK_SIZE" help:"maximum chunk size" default:"20" optional:""`
+	Template string `name:"template" short:"t" env:"DS_TEMPLATE_FILE" help:"transformation template file path" type:"path" optional:""`
 }
 
-func (t *TransformCmd) Run(context *kong.Context) error {
-	content, err := t.getTemplateContent()
+func (t *TransformCmd) Run(kongContext *kong.Context) error {
+	template, err := t.getTemplateContent()
 	if err != nil {
 		return err
 	}
-	return plugin.NewDSPlugin(plugin.WithTemplate(content), plugin.WithMaxChunkSize(t.MaxChunkSize)).Transform()
+	timeoutCtx, cancel := context.WithTimeout(context.Background(), 1500*time.Millisecond)
```

Comment:
Instead of creating background contexts on the fly throughout the code, it would be better to create a single context in `main` and pass it down when running commands.
We use that pattern in all our CLIs.

Here's an example from topaz:
1. We pass an object with info needed by all commands as an argument to `kong.Run`: https://github.com/aserto-dev/topaz/blob/main/cmd/topaz/main.go#L36
2. Our commands can then use it as an argument: https://github.com/aserto-dev/topaz/blob/main/pkg/cli/cmd/configure.go#L23C32-L23C32

Reply:
But this can happen in a separate PR. This one is big enough

Reply:
Can I also wrap this `CommonContext` in a `context.WithTimeout`?

Reply:
Yeah, of course. It would simply contain a `context.Context` that can be used in any context-aware operation.